tests/cypress/tests-e2e/embed.cy.js

```diff
@@ -34,7 +34,7 @@ describe('Testing the embed view', () => {
             .should(
                 'have.attr',
                 'href',
-                `#/map?lang=en&center=2660000,1190000&z=1&bgLayer=test.background.layer2&topic=ech&layers=`
+                `#/map?lang=en&center=2660000,1190000z=1&bgLayer=test.background.layer2&topic=ech&layers=`
```

Comment:
You probably don't want this changes, this make your test failing

Reply:
good catch